### ISSUE TITLE
remove translations from components via API

### ISSUE THREAD

**Issue by eighthave** [WeblateOrg/weblate#2724]
**Is your feature request related to a problem? Please describe.**

A new admin enabled the "Language Consistency" plugin, which filled all the components with empty translations.  This makes it harder to manage, especially with the squash/rebase workflow.

**Describe the solution you'd like**

A way to use the existing _remove_ function via the API.  E.g. https://hosted.weblate.org/remove/f-droid/website-tutorials/eu/

The API could use the same confirmation string, e.g. the whole project/component/language triplet.

**Describe alternatives you've considered**

It is possible via the web UI, but that is really time consuming to do for more than a few translations.

**Additional context**

Here's the script that I wrote to do this, it is only lacking the API call to do the actual remove:

```python
import os
import requests
import time

components_url = 'https://hosted.weblate.org/api/projects/f-droid/components/'
translations_url = 'https://hosted.weblate.org/api/components/f-droid/%s/translations/'

r = requests.get(components_url)
r.raise_for_status()
data = r.json()

components = []
for component in data['results']:
    components.append(component['slug'])

deleteme = dict()
for component in components:
    r = requests.get(translations_url % component)
    r.raise_for_status()
    data = r.json()
    for result in data['results']:
        if result['translated'] == 0 \
           and result['translated_words'] == 0 \
           and result['translated_percent'] == 0:
            language_code = result['language_code']
            if language_code not in deleteme:
                deleteme[language_code] = []
            deleteme[language_code].append(component)

for language, components in deleteme.items():
    for component in sorted(components):
        os.system('firefox https://hosted.weblate.org/projects/f-droid/%s/%s/#delete'
                  % (component, language))
```

**Comment by eighthave** [comment]
I suppose this could be solved via a "remove empty translations" addon too.  That would work for me as well.

**Comment by nijel** [comment]
I think this is better suited for API extension than an addon.

**Comment by eighthave** [comment]
FYI, it is related to #2711

**Comment by nijel** [comment]
Thank you for your report, the issue you have reported has been fixed by f638ca61402fb3572adacc66a8430891225ec273.

* In case you see problem with the fix, please comment on this issue. 
* In case you see similar problem, please open separate issue.
* If you are happy with the outcome, consider supporting Weblate by [donating](https://weblate.org/donate/).
